modules/publisher/api.py

```diff
@@ -127,18 +108,8 @@ def get_account(session):
         headers=headers
     )
 
-    verified_response = verify_response(
-        response,
-        session,
-        ACCOUNT_URL,
-        '/account',
-        '/login',
-    )
-
-    if verified_response is not None:
-        return {
-            'redirect': verified_response
-        }
+    if authentication.verify_headers(response.headers):
```

Comment:
This function name is pretty confusing but that may be a subject for a different PR